### PR TITLE
ci(github): transport e2e test to github

### PR DESCRIPTION
**Pull request by karliatto**
## Description

Migration of Transport e2e tests to GitHub Actions.

Related to https://github.com/trezor/trezor-suite/issues/7916

It works!

image

### CODE REVIEW

File: .github/workflows/connect-transport-e2e-test.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Transport E2E Tests
```
Comment (vdovhanych) [inline .github/workflows/connect-transport-e2e-test.yml]: You should adhere to the naming convention.

File: .github/workflows/connect-transport-e2e-test.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - "packages/protobuf/**"
+      - "packages/protocol/**"
+      - "packages/trezor-user-env-link/**"
+      - "packages/utils/**"
+      - "docker/docker-compose.transport-test.yml"
+      - ".github/workflows/connect-transport-e2e-test.yml"
+      - "yarn.lock"
+
+jobs:
+  e2e:
```
Comment (vdovhanych) [inline .github/workflows/connect-transport-e2e-test.yml]: this should be more descriptive

**Review by vdovhanych — COMMENTED** [review]
Just small nitpicks

File: .github/workflows/connect-transport-e2e-test.yml (hunk 1 above)
Comment (karliatto) [reply .github/workflows/connect-transport-e2e-test.yml]: :+1: https://github.com/trezor/trezor-suite/pull/11609/commits/b52b5e0dd372879acf671166504ef658cfc20cbd

File: .github/workflows/connect-transport-e2e-test.yml (hunk 2 above)
Comment (karliatto) [reply .github/workflows/connect-transport-e2e-test.yml]: Something like: https://github.com/trezor/trezor-suite/pull/11609/commits/b52b5e0dd372879acf671166504ef658cfc20cbd  :question:

File: .github/workflows/connect-transport-e2e-test.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+jobs:
+  transport-e2e:
+    runs-on: ubuntu-latest
+    env:
+      COMPOSE_FILE: ./docker/docker-compose.transport-test.yml
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          lfs: true
```
Comment (vdovhanych) [inline .github/workflows/connect-transport-e2e-test.yml]: do we need lfs here?

Comment (karliatto) [reply .github/workflows/connect-transport-e2e-test.yml]: We do not need, updated! Thanks!

**Review by vdovhanych — APPROVED** [review]

**Pull request merged**